pytest_mongodb/plugin.py

```diff
@@ -82,7 +82,9 @@ def clean_database(db):
 
 
 def load_fixtures(db, config):
-    basedir = config.getoption('mongodb_fixture_dir') or config.getini('mongodb_fixture_dir')
+    fixture_dir = config.getoption('mongodb_fixture_dir') or config.getini('mongodb_fixture_dir')
+    if not os.path.isabs(fixture_dir):
+        basedir = str(config.inifile.join('..', fixture_dir))
```

Comment:
What if `fixture_dir` actually **is** an absolute path. In that case basedir will never be set. Additionally `config.rootdir` is probably what you're looking for. I would suggest the following implementation.
```python
basedir = config.getoption('mongodb_fixture_dir') or config.getini('mongodb_fixture_dir')
if not os.path.isabs(basedir):
    basedir = config.rootdir.join(basedir).strpath
```

Reply:
Yes, I forgot about this case. :disappointed: Corrected with your suggestion, thank you for your review.